index.html

```diff
@@ -11,86 +11,139 @@
+				<img class="desk" src="img/home/home-img-2.png">
+				<img class="mob" src="img/home/home-mobile-img-2.png">
+			</div>
+			<div class=block2-right>
+				<h2>Futuristic Designs</h2>
+				<p>Et sed autem causae appareat, tempor abhorreant te mei, facer facilisis sit ea. Eu timeam vidisse consectetuer sed. Duo etiam laboramus dissentiet in, nec no errem</p> 
+				<div class="btn">View Designs</div>
+			</div>
+		</section>
+		<div class="h2Header"><h2>Recent Projects</h2></div>
+		<section class="projects theVillas">
+			<div>
+				<img class="desk" src="img/home/home-villas-img.png">
+				<img class="mob"src="img/home/home-mobile-villas-img.png">
+				<div class="top">THE VILLAS</div>
+			</div>
+			<p>The Villas bring to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p>
+
+			<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.</p>
+		</section>
+		<section class="projects outskirts">
+			<div>
+				<img class="desk" src="img/home/home-outskirts-img.png">
+				<img class="mob" src="img/home/home-mobile-outskirts-img.png">
+				<div class="top">OUTSKIRTS</div>
+			</div>
+				<p>The Outskirts are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p>
+
+				<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.</p>
+		</section>
+		<section class="projects theBlocks">
+			<div>
+				<img class="desk" src="img/home/home-the-blocks-img.png">
+				<img class="mob" src="img/home/home-mobile-the-blocks-img.png">
+				<div class="top">THE BLOCKS</div>
+			</div>
+				<p>The Blocks are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p> 
+
+				<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.</p>
+		</section>
+	</div> <!-- Main Content End -->
+	<!-- Home Page Content End -->
+
+
+	<!-- FOOTER CONTENT -->
+	<div class="footer">
+		<footer>
```

Comment:
You generally don't need a div wrapped around a footer tag, you can just use and target the footer.